plugins/com.google.cloud.tools.eclipse.appengine.localserver/META-INF/MANIFEST.MF

```diff
@@ -33,4 +34,5 @@ Require-Bundle: org.eclipse.wst.server.core;bundle-version="1.7.0",
  org.eclipse.ui.console;bundle-version="3.6.100",
  org.eclipse.ui.ide;bundle-version="3.11.0",
  com.google.cloud.tools.eclipse.appengine.ui;bundle-version="0.1.0"
-Export-Package: com.google.cloud.tools.eclipse.appengine.localserver.runtime
+Export-Package: com.google.cloud.tools.eclipse.appengine.localserver.runtime,
```

Comment:
It's an autofix suggested by Eclipse when the relevant classes couldn't be found by the test plugin. I think if we don't export them, the test plugin can't see them, even though they'e in the same package.